src/cms/page-scheduled-changes.md

```diff
@@ -7,3 +7,21 @@ Page changes can be applied on schedule, and grouped with other content changes.
 
 ![The Home page displays scheduled changes at the top]({% link images/images/page-scheduled-change.png %}){: .zoom}
 _Scheduled changes for the Home page_
+
+## Schedule an update to a page
```

Comment:
This procedure is a repeat of the one in https://docs.magento.com/user-guide/cms/content-staging-scheduled-update.html#schedule-an-update-to-a-product. Rather than create redundant content, let add a specific link to that page/section.

Reply:
Done, thank you!